ArmPlatformPkg/MemoryInitPei/MemoryInitPeiLib.c

```diff
@@ -108,14 +108,69 @@ MemoryPeim (
 
   if (!Found) {
     // Reserved the memory space occupied by the firmware volume
-    BuildResourceDescriptorV2 (
-      EFI_RESOURCE_SYSTEM_MEMORY,
-      ResourceAttributes,
-      PcdGet64 (PcdSystemMemoryBase),
-      PcdGet64 (PcdSystemMemorySize),
-      EFI_MEMORY_WB,
-      NULL
-      );
+    // MU_CHANGE: Carve out MM communication buffer from system memory
+    // But pay attention to the potential overlap with the mm communication buffer
+    if (PcdGet64 (PcdMmBufferBase) >= PcdGet64 (PcdSystemMemoryBase) &&

```

Comment:
There are a lot of PcdGet64 for the same value, for readability you could do those ahead of this

Reply:
Agreed, will update.

Reply:
This is updated.